components/layout/NavBar.vue

```diff
@@ -1,6 +1,7 @@
 <script setup lang="ts">
-import Luigi from '~/assets/luigi.svg';
+import Luigi from 'assets/luigi.svg';
```

Comment:
What was the motivation behind this change?

Reply:
there wasn't one tbh, can't remember why this ended up changing